PyLTSpice/raw/raw_read.py

```diff
@@ -358,7 +358,7 @@ class RawRead(object):
         'Frequency Response Analysis',
     )
 
-    def __init__(self, raw_filename: str, traces_to_read: Union[str, List[str], Tuple[str], None] = '*', **kwargs):
+    def __init__(self, raw_filename: str, traces_to_read: Union[str, List[str], Tuple[str, ...], None] = '*', **kwargs):
```

Comment:
Thanks for correcting this.
Do you know if the typing. List also has to have the same pattern.
The https://docs.python.org/3/library/typing.html says nothing in that regard.

Reply:
I wondered about that too, I didn't see it mentioned in the documentation.